src/codeenginestudio/thuy/Exercise1.java

```diff
@@ -1,23 +1,30 @@
 package codeenginestudio.thuy;
 
+import java.util.Arrays;
+
 public class Exercise1 {
 
     public static void main(String args[]) {
 
-        String s1 = "is2 Thi1s T4est 3a";
+        String s1 = "is2 Thi1s T4est 3a thu hoa";
```

Comment:
Initially, I thought the input should follow the rules. Example: ""is2 Thi1s first4 J5avaT6est 3the"

Reply:
I got it. This case "is2 Thi1s first4 J5avaT6est 3the"  I passed..
But I am trying with other cases